utils/data-migration/migrate-acbs-utilisation-data-to-tfm/index.ts

```diff
@@ -0,0 +1,61 @@
+import { SqlDbDataSource } from '@ukef/dtfs2-common/sql-db-connection';
+import { FacilityUtilisationDataEntity, ReportPeriodPartialEntity } from '@ukef/dtfs2-common';
+import { DataSource } from 'typeorm';
+import { existsSync, readFileSync } from 'fs';
+import { resolve } from 'path';
+
+const JSON_FILE_PATH = resolve(__dirname, 'acbs-utilisation-data.json');
+
+const REPORT_PERIOD: ReportPeriodPartialEntity = {
+  start: { month: 1, year: 2020 },
+  end: { month: 3, year: 2020 },
+};
+
+type UtilisationDataEntry = {
+  facilityId: string;
+  utilisation: number;
+  fixedFeePremium: number;
+};
+
+const validateReportPeriod = (reportPeriod: ReportPeriodPartialEntity): void => {
+  const { start, end } = reportPeriod;
+  if (start.year > end.year || (start.year === end.year && start.month > end.month)) {
```

Comment:
Can this please be refactored to two conditional variables.

```javascript
const cond1 = ..
const cond2 = ..

if (cond1 || cond2) { ...
```

Reply:
Made this change, thanks for spotting forgot to pull these out